.github/workflows/build_and_push_sdk_packages.yaml

```diff
@@ -4,22 +4,38 @@ on:
   push:
     tags:
     # Not the right semver regexp, but good enough
-      - 'v[0-9]+.[0-9]+.[0-9]+-*'
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
❓ Would it make sense for us to prefix SDK on SDK tags? e.g. `sdk/v0.1.0`? 

I'm trying to figure out if this might help us out later when we come to decide how we might handle versions independently (SDK, API, backend, etc.).

Reply:
I think it doesn't pay off at this stage (as you mention, we're currently in lockstep). When we finally decide on the version handling for independent components, then we will implement the appropriate mechanism.